vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.jsp

```diff
@@ -92,93 +171,143 @@ $(document).ready(function() {
 
 });
 </script>
-
 <ol class="breadcrumb">
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/dashboard" />">Dashboard</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/list" />">Modules</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/${module.id}" />">${module.name}</a></li>
-  <li class="breadcrumb-item active">${slide.name}</li>
+    <li class="breadcrumb-item"><a
+        href="<c:url value="/staff/dashboard" />"
+    >Dashboard</a></li>
+    <li class="breadcrumb-item"><a
+        href="<c:url value="/staff/module/list" />"
+    >Modules</a></li>
+    <li class="breadcrumb-item"><a
+        href="<c:url value="/staff/module/${module.id}" />"
+    >${module.name}</a></li>
+    <li class="breadcrumb-item active">${slide.name}</li>
 </ol>
-
-<h1>Slide: ${slide.name}</h1>
+<div class="error"></div>
+<!-- title -->
+<div class="row align-items-center">
+    <h1 id="title" style="margin-bottom: 0%; margin-left: 1%;">Slide: ${slide.name}</h1>
+    <a id="editTitle" class="btn" href="#"
+        style="float: left; margin-right: 1%;"
+    ><i class="fas fa-edit"></i></a>
+    <button id="submitTitle" type="button"
+        class="btn btn-outline-success"
```

Comment:
let's use the button type that is used for all the other buttons as well (the green really doesn't work here..)